### ISSUE TITLE
Unknown column 'A.t3ver_move_id' in 'where clause'

### ISSUE THREAD

**Issue by tomhatzer** [fnagel/t3extblog#198]
Typo3 version: 8.7.15
PHP version: 7.2.2
MySQL version: 5.5.5 (MariaDB 10.2.14)
Activated extensions causing trouble: version / workspaces

I'm receiving the following error message when I'm going into the "Workspaces" tab of my Typo3 backend.

**Error Message:**
```
Core: Exception handler (WEB): Uncaught TYPO3 Exception: An exception occurred while executing 'SELECT `A`.`pid` AS `wspid`, `B`.`uid` AS `t3ver_oid`, `C`.`uid` AS `uid`, `B`.`pid` AS `livepid` FROM `tx_t3blog_cat` `A`, `tx_t3blog_cat` `B`, `tx_t3blog_cat` `C` WHERE (`A`.`t3ver_state` = ?) AND (`B`.`pid` > ?) AND (`B`.`t3ver_state` = ?) AND (`B`.`t3ver_wsid` = ?) AND (`C`.`pid` = ?) AND (`C`.`t3ver_state` = ?) AND (`A`.`t3ver_move_id` = `B`.`uid`) AND (`B`.`uid` = `C`.`t3ver_oid`) AND (`A`.`t3ver_wsid` = ?) AND (`C`.`t3ver_wsid` = ?) AND (`A`.`pid` IN (?, ?)) AND ((`A`.`deleted` = 0) AND (`B`.`deleted` = 0) AND (`C`.`deleted` = 0)) ORDER BY `A`.`uid` ASC' with params ["3", 0, "0", 0, -1, "4", 0, 0, 1, 1]: Unknown column 'A.t3ver_move_id' in 'where clause'
```

A quick fix for this is adding the column **t3ver_move_id** to table **tx_t3blog_cat** manually. Afterwards it works as expected again.

**Comment by tomhatzer** [comment]
The field is also missing in the following two tables:

- tx_t3blog_post
- tx_t3blog_blogroll

I've added them to my PR too.

**Comment by fnagel** [comment]
Thanks for your contribution!

Sounds legit. I must confess I've never tested the extension with versioning. Any feedback / notes if this works without issues? 

PR looks good except for the commit messages. Please use something like `[TASK|BUGFIX|FIX|FEATURE|DOC] What has been fixed`

**Comment by fnagel** [comment]
Merged with #199
